### PR TITLE
More german translations

### PR DESCRIPTION
**Pull request by jammon**

### CODE REVIEW

File: locales/de/translation.json [hunk 1]
```diff
@@ -89,8 +91,11 @@
   "we_believe_in_open_source": "Wir glauben an Open Source",
   "index_page_open_source": "Syncloud wurde basierend auf bekannten Open-Source-Technologien wie Debian, Avahi, Python, nginx und vielen weiteren gebaut. Syncloud wäre ohne diese großartigen Projekte nicht möglich gewesen. Wir glauben daran, dass Open-Source-Technologien mehr Potenzial haben und eine bessere Qualität und Sicherheit bieten, weshalb wir auch Syncloud als Open-Source-Projekt aufgesetzt haben. Forken Sie uns auf GitHub!",
   "contact_us": "Kontaktieren Sie uns",
+// FAQ
```
Comment (cyberb) [inline locales/de/translation.json]: json does not support comments, hence Contributor_1, Contributor_2 ...

File: locales/de/translation.json [hunk 2]
```diff
@@ -103,5 +108,11 @@
...
+  "faq_7_answer": "Syncloud speichert alle Daten entweder direkt auf Ihrem Syncloud-Gerät oder auf einer angeschlossenen externen Festplatte. Dies beinhaltet auch alle Daten der Syncloud-Anwendungen. Da Syncloud auch freie Gigabyte auf einer SD-Karte oder einem Flash-Speicher zum Ablegen der Daten verwenden kann, ist es für die Verwendung von Syncloud nicht zwingend notwendig eine externe Festplatte zu verwenden.",
+  "faq_8": "Kann ich mit SSH auf mein Gerät zugreifen?",
+  "faq_8_answer": "Ja. Die Standard-Anmeldedaten vor der Aktivierung sind Login: root, Passwort: syncloud. Nach der Aktivierung ist das Passwort identisch mit dem Gerätepasswort.",
+  "faq_9": "Kann ich mein Gerät auch ohne mobile Anwendung aktivieren?",
+  "faq_9_answer": "Ja. Sie müssen die Geräte-IP herausfinden (normalerweise auf Ihrer Router-Webseite), dann öffnen Sie http://&lt;IP&gt; in Ihrem Browser. Dann durchlaufen Sie die Aktivierungsprozedur.",
```
Comment (cyberb) [inline locales/de/translation.json]: 1. i18next escapes special symbols, so no need to escape: http://&lt;IP&gt; => http://<IP>;
2. Also why do you need a semicolon at the end?

Comment (jammon) [reply locales/de/translation.json]: 1. Thank you, I didn't know that. I fixed this.
2. There were no semicolons, just the html entities.

**Comment by cyberb** [comment]
pushed to syncloud.org

**Pull request merged**